src/main/java/org/isf/medicalstock/manager/MovStockInsertingManager.java

```diff
@@ -62,7 +62,7 @@ public MovStockInsertingManager(MedicalStockIoOperations medicalStockIoOperation
 	 * @param checkReference - if {@code true} it will use {@link #checkReferenceNumber(String) checkReferenceNumber}
 	 * @throws OHServiceException
 	 */
-	protected void validateMovement(Movement movement, boolean checkReference) throws OHServiceException {
+	protected void validateMovement(Movement movement, boolean checkReference, boolean checkWard) throws OHServiceException {
```

Comment:
this change will affect also API... why do you need to exclude the check on the Ward field?